build/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/mattermost/mattermost-plugin-starter-template/build
 go 1.12
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200723144633-ed34468996e6
+	github.com/mattermost/mattermost-server/v6 v6.0.0-20210824183758-eaa37b52ee5a
```

Comment:
Isn't this commit prior to the latest changes?

Reply:
Good catch :+1: Fixed.